src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin: 0 4px 0 0;
```

Comment:
not all the stars need margin 😉

<img width="224" alt="image" src="https://user-images.githubusercontent.com/55802630/172790621-287a8d4f-7c87-4a2c-82f9-cf8b40c0d1b7.png">

Reply:
added 'gap' instead of 'margin'
is it ok? :-)

Reply:
of course 😉